### PR TITLE
Bugfix/sp 274

### PR DESCRIPTION
**Pull request by mpiroh**

### CODE REVIEW

File: src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js [hunk 1]
```diff
@@ -97,11 +97,11 @@ const ActiveEmployeesTable = ({
         checkFilterValues(filterInputs.officeLocation, empl.officeLocation) &&
         checkFilterValues(
           filterInputs.year,
-          empl.endDate ? empl.endDate[0] : null
+          empl.entryDate ? empl.entryDate[0] : null
```
Comment (steeveam) [inline src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js]: endDate into entryDate?

File: src/components/AllEmployees/ActiveEmployeesGrid.js [hunk 2]
```diff
@@ -99,11 +99,11 @@ const ActiveEmployeesGrid = ({
       checkFilterValues(filterInputs.officeLocation, empl.officeLocation) &&
       checkFilterValues(
         filterInputs.year,
-        empl.endDate ? empl.endDate[0] : null
+        empl.entryDate ? empl.entryDate[0] : null
       ) &&
       checkFilterValues(
         filterInputs.month,
-        empl.endDate ? empl.endDate[1] : null
+        empl.entryDate ? empl.entryDate[1] : null
```
Comment (steeveam) [inline src/components/AllEmployees/ActiveEmployeesGrid.js]: endDate into entryDate?

File: src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js [hunk 3]
```diff
@@ -97,11 +97,11 @@ const ActiveEmployeesTable = ({
...
         ) &&
         checkFilterValues(
           filterInputs.month,
-          empl.endDate ? empl.endDate[1] : null
+          empl.entryDate ? empl.entryDate[1] : null
```
Comment (steeveam) [inline src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js]: endDate into entryDate?

File: src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js (hunk 1 above)
Comment (mpiroh) [reply src/components/AllEmployees/EmployeesTable/ActiveEmployeesTable.js]: Yes, it is ActiveEmployeesTable, so year/month should filter according to entry date (those employees don't even have end dates)

**Pull request merged**